### PR TITLE
ir: Do not create new notary requests for SN netmap entrances

### PR DESCRIPTION
**Pull request by carpawell**
See #2612, https://github.com/nspcc-dev/neofs-contract/pull/324. Refs #830.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Changelog for NeoFS Node
 
 ### Changed
 - Created files are not group writable (#2589)
+- IR does not create new notary requests for the SN's bootstraps but resigns the received ones instead (#2717)
```
Comment (roman-khimov) [inline CHANGELOG.md:18]: signs

**Review by roman-khimov — APPROVED** [review]
It certainly fixes #830. Contracts can be improved, but gradually and in a separate issue, we need to ensure only one method is called for at least a release and then we can remove the other one.

File: pkg/innerring/processors/netmap/process_peers.go [hunk 2]
```diff
@@ -62,26 +61,9 @@ func (np *Processor) processAddPeer(ev netmapEvent.AddPeer) {
...
-			originalRequest.MainTransaction.Nonce,
-			nil,
-			methodAddPeerNotary,
-			nodeInfoBinary,
-		)
-
+		err = np.netmapClient.Morph().NotarySignAndInvokeTX(ev.NotaryRequest().MainTransaction)
 		if err != nil {
-			np.log.Error("can't invoke netmap.AddPeer", zap.Error(err))
+			np.log.Error("can't sign notary netmap.AddPeer", zap.Error(err))
```
Comment (cthulhu-rider) [inline pkg/innerring/processors/netmap/process_peers.go:66]: `send` matters too
```suggestion
			np.log.Error("can't sign and send notary request calling netmap.AddPeer", zap.Error(err))
```

**Review by cthulhu-rider — APPROVED** [review]
i'd prefer to not embody commit by issues only. Text and other commit refs are more reliable, issue refs complement

File: pkg/innerring/processors/netmap/process_peers.go [hunk 3]
```diff
@@ -62,26 +61,9 @@ func (np *Processor) processAddPeer(ev netmapEvent.AddPeer) {
...
-		err = np.netmapClient.Morph().NotaryInvoke(
-			np.netmapClient.ContractAddress(),
-			0,
-			originalRequest.MainTransaction.Nonce,
-			nil,
-			methodAddPeerNotary,
-			nodeInfoBinary,
-		)
-
+		err = np.netmapClient.Morph().NotarySignAndInvokeTX(ev.NotaryRequest().MainTransaction)
```
Comment (cthulhu-rider) [inline pkg/innerring/processors/netmap/process_peers.go:64]: btw can we use `originalRequest` here? code will reflect "original request" processing more clear

Comment (carpawell) [reply pkg/innerring/processors/netmap/process_peers.go:64]: i do not see any api about signing notary requests in `neo-go`. it is always TXs

Comment (cthulhu-rider) [reply pkg/innerring/processors/netmap/process_peers.go:64]: i mean we have https://github.com/nspcc-dev/neofs-node/blob/de84b12e02dc6fe5bc564928d74398b2a32e26d6/pkg/innerring/processors/netmap/process_peers.go#L21-L22

they are the same as `ev.NotaryRequest()` right? seems like that. If so, it's worth to pass `tx` or `originalRequest.MainTransaction` them as method arg

**Pull request merged**